design/Proposal For View Definition.md

```diff
@@ -0,0 +1,84 @@
+﻿
+
+
+
+## Proposal to extend the View Definition
+**Background**
+The current  interface for a SqlView  which is a first class object in Nessie specifies a Sql text and a Dialect  and schema :
+
+    public interface SqlView extends Contents {  
+      
+      public static enum Dialect {  
+        HIVE,  
+        SPARK,  
+        DREMIO,  
+        PRESTO  
+      }  
+      
+      String getSqlText();  
```

Comment:
I guess it's the query? any restrictions to be put on it? For example, all table reference should be absolute (and not relative to a default schema?)